students/ShinTran/lesson04/mailroom2.py

```diff
@@ -0,0 +1,137 @@
+'''
+Shin Tran
+Python 210
+Lesson 4 Assignment
+'''
+
+master_list = [] # For all donations made
+name_list = [] # For list of unique names
+
+
+# Creates the master_list where it has 12 donations pre-populated
+def initialize():
+    global master_list
+    master_list = [
+        ["Jennifer Miller", 91561.25],
+        ["Mary Johnson", 5811.05],
+        ["William Rodriguez", 41113.42],
+        ["Patricia Brown", 19184.81],
+        ["William Rodriguez", 51227.53],
+        ["Jennifer Miller", 53514.94],
+        ["Michael David", 31051.46],
+        ["Mary Johnson", 71646.16],
+        ["William Rodriguez", 69244.21],
+        ["Mary Johnson", 12689.07],
+        ["Michael David", 57145.50],
+        ["Patricia Brown", 81679.15],
+    ]
```

Comment:
in lesson4 this should be using a dictionary data structure; example: `donors = {"Jenn": [999], "Will": [30, 20]}`